templates/zerver/help/manage-inactive-streams.md

```diff
@@ -1,9 +1,9 @@
 # Manage inactive streams
 
-In normal Zulip usage, streams fall in and out of use. By default,
-Zulip automatically moves streams that haven't been used in a while to
-the bottom of your streams list in the left sidebar.  This helps
-prevent low-traffic streams from cluttering the left sidebar.
+In normal Zulip usage, streams fall in and out of use. By default, Zulip
+automatically moves streams that haven't been used in a while to the bottom
+of your streams list in the left sidebar under the section labeled **Inactive**.
+This helps prevent low-traffic streams from cluttering the left sidebar.
```

Comment:
How about something like this?

> By default, Zulip
automatically moves streams that haven't been used in a while to the **Inactive** section at the bottom
of your streams list in the left sidebar.